src/components/productsList/ProductsList.component.tsx

```diff
@@ -0,0 +1,28 @@
+import type { FC } from 'react';
+
+import { products } from '@/moc/moc';
+
+import ProductCard from '../productCard/ProductCard.component';
+
+import styles from './productsList.module.css';
+
+interface Props {
```

Comment:
all `interfaces` must be in another folder
Check it please in each component of project

Reply:
Done.